### PR TITLE
CLI set up for running prediction using a previous model

### PR DESCRIPTION
**Pull request by ecpierce**
main changes:
-added notes to readme about what has changed and how to run
- made new scripts to run separate module to do prediction if you already have a model, and edited bsoid_app/predict.py to work with this

### CODE REVIEW

File: README.md
Comment (taylorreiter) [inline README.md]: wait im confused. the user has to go into those python scripts to look at the documentation? would it be simple to put that in the readme? if not, can you at least link to the py files?

**Review by taylorreiter — COMMENTED** [review]
lgtm, a couple questions about the readme, but again will wait until Mert's approval! congrats on getting it this far!

File: bsoid_predict.py
Comment (mertcelebi) [inline bsoid_predict.py:9]: You could probably move this to a `utils.py` and import it in both places to reduce duplicaion

File: bsoid_predict.py
Comment (mertcelebi) [inline bsoid_predict.py]: You probably should use the ruff VSCode extension to lint these type of lines

File: bsoid_app/predict.py
Comment (mertcelebi) [inline bsoid_app/predict.py:34]: `FTYPE` was an array but here is a string. Will this cause issues? Or do you need smth like `self.filetype = [FTYPE]`?

File: bsoid_app/predict.py
Comment (mertcelebi) [inline bsoid_app/predict.py]: Yeah line 34 won't work

**Review by mertcelebi — CHANGES_REQUESTED** [review]
Some small fixes

File: README.md
Comment (ecpierce) [reply README.md]: added to readme

File: bsoid_predict.py
Comment (ecpierce) [reply bsoid_predict.py]: reformatted for readability

File: bsoid_app/predict.py
Comment (ecpierce) [reply bsoid_app/predict.py:34]: think fine as string, originally it is a dropdown selection, but there is only one ftype

File: bsoid_app/predict.py
Comment (ecpierce) [reply bsoid_app/predict.py]: removed these lines, we don't need them. self.filetype is defined in setup

File: bsoid_predict.py
Comment (ecpierce) [reply bsoid_predict.py:9]: opting not to do this

**Review by taylorreiter — COMMENTED** [review]
Just commenting that I have nothing to add...congrats on getting it this far!

**Review by mertcelebi — APPROVED** [review]

**Pull request merged**
